packages/ui-navigation/src/Navigation/NavigationItem/index.tsx

```diff
@@ -67,7 +75,7 @@ class NavigationItem extends Component<NavigationItemProps> {
     this.props.makeStyles?.({ minimized: this.props.minimized })
   }
 
-  renderLink() {
+  renderLink(addRef: boolean) {
```

Comment:
It took some time for me to understand this. How about putting hasTooltip into a getter so you can use it here and in the render too? For first read it is not obvious what the "addRef" depends on.

Reply:
hmm I like it better this way, getters are hard to read IMO